editor.planx.uk/src/@planx/components/DateInput/Editor.test.tsx

```diff
@@ -51,7 +51,7 @@ describe("DateInputComponent - Editor Modal", () => {
 
     expect(await screen.findByText(minError)).toBeVisible();
     expect(await screen.findByText(maxError)).toBeVisible();
-  });
+  }, 10_000);
```

Comment:
Quite a few of these needed to be bumped - it seems as vitest is getting more mature it's also slowing down 🫤

This issues could well be related to jsdom / MUI / RTL / formik of course, but at some point we're going to need to spend a bit of time I think trying to resolve these issues as speed things up (at least on CI).